web/app.js

```diff
@@ -2187,7 +2187,11 @@ function webViewerInitialized() {
     appConfig.mainContainer.addEventListener("dragover", function (evt) {
       evt.preventDefault();
 
-      evt.dataTransfer.dropEffect = "move";
+      evt.dataTransfer.dropEffect =
+        evt.dataTransfer.effectAllowed === "all" ||
+        evt.dataTransfer.effectAllowed === "uninitialized"
+          ? "move"
+          : evt.dataTransfer.effectAllowed;
```

Comment:
Let's reduce some unnecessary repetition here, by introducing a temporary variable:
```javascript
const { effectAllowed } = evt.dataTransfer;
evt.dataTransfer.dropEffect =
  effectAllowed === "all" || effectAllowed === "uninitialized"
  ? "move"
  : effectAllowed;
```

Reply:
Wait, this will now also support "none" which would thus break drag-and-drop here. I don't think that we actually want that, so based on https://github.com/mozilla/pdf.js/issues/14942#issuecomment-1132715355 it seems that we should perhaps just special-case the "copy"-case instead:
```javascript
evt.dataTransfer.dropEffect =
  evt.dataTransfer.effectAllowed  === "copy" ? "copy" : "move";
```

Reply:
applied in [b1941f8](https://github.com/mozilla/pdf.js/pull/14948/commits/b1941f89ea09cbdb10a281926ee8351bf7faf62e)

Reply:
Thank you; please also squash the commits, see https://github.com/mozilla/pdf.js/wiki/Squashing-Commits

Reply:
yep, squashed